apps/android/app/src/main/java/chat/simplex/app/model/NtfManager.kt

```diff
@@ -218,19 +218,15 @@ class NtfManager(val context: Context, private val appPreferences: AppPreference
   }
 
   private fun hideSecrets(cItem: ChatItem) : String {
-    val md = cItem.formattedText
-    return if (md == null) {
-      if (cItem.content.text != "") {
-        cItem.content.text
-      } else {
-        if (cItem.content.msgContent is MsgContent.MCVoice) generalGetString(R.string.voice_message) else cItem.file?.fileName ?: ""
-      }
-    } else {
+    val md = cItem.formattedText_
```

Comment:
this text_ is completely unnecessary because it simply hides a real condition (meta.itemDeleted) behind == null condition, it's not reducing complexity and only creates an unnecessary indirection.

Reply:
it only imitates original formatted text - the fact that it could be null